sklearn/tree/_tree.pyx

```diff
@@ -1106,24 +1088,24 @@ cdef class Tree:
                     left = &nodes[node.left_child]
                     right = &nodes[node.right_child]
 
-                    importance_data[node.feature] += (
+                    importances[node.feature] += (
                         node.weighted_n_node_samples * node.impurity -
                         left.weighted_n_node_samples * left.impurity -
                         right.weighted_n_node_samples * right.impurity)
                 node += 1
 
-        importances /= nodes[0].weighted_n_node_samples
+        importances = np.divide(importances, nodes[0].weighted_n_node_samples)
 
         if normalize:
             normalizer = np.sum(importances)
 
             if normalizer > 0.0:
                 # Avoid dividing by zero (e.g., when root is pure)
-                importances /= normalizer
+                importances = np.divide(importances, normalizer)
 
-        return importances
+        return np.asarray(importances)
 
-    cdef cnp.ndarray _get_value_ndarray(self):
+    cdef DOUBLE_t[:, :, ::1] _get_value_ndarray(self):
```

Comment:
Given the naming of `_get_value_ndarray`, I prefer it to return a `ndarray` instead of a memoryview.